src/openrct2-ui/windows/ViewClipping.cpp

```diff
@@ -430,14 +430,14 @@ static void window_view_clipping_paint(rct_window* w, rct_drawpixelinfo* dpi)
                 // Print the value in the configured measurement units.
                 switch (gConfigGeneral.measurement_format)
                 {
-                    case MEASUREMENT_FORMAT_METRIC:
-                    case MEASUREMENT_FORMAT_SI:
+                    case MeasurementFormat::Metric:
+                    case MeasurementFormat::SI:
                         clipHeightValueInMeters = static_cast<fixed32_2dp>(
                             FIXED_2DP(gClipHeight, 0) / 2 * 1.5f - FIXED_2DP(10, 50));
                         gfx_draw_string_left(
                             dpi, STR_UNIT2DP_SUFFIX_METRES, &clipHeightValueInMeters, w->colours[0], screenCoords);
                         break;
-                    case MEASUREMENT_FORMAT_IMPERIAL:
+                    case MeasurementFormat::Imperial:
                     default:
```

Comment:
You can probably (and should) remove this `default` too, as the switch handles all cases.